src/app/App.js

```diff
@@ -72,10 +73,40 @@ export const App = ({ options }) => {
   });
 
   // Modal window
-  ModalWindow(app);
+  const modalWindow = ModalWindow(app);
   // Modal window methods
   // modalWindow.show(Here_some_content_as_HTMLElement);
   // modalWindow.close  to close the window and remove the content
+
+  const testButton = render({
+    component: Button({
+      id: 'abcd',
+      btnText: 'TEST BUTTON',
+      classList: [''],
+      onClickFn: testFunction,
+    }),
+    inside: app,
+  });
+
+  function testFunction() {
+    const contemodalWindowContent = ModalWindowContent(
+      [
+        { id: 1, isCorrect: true },
+        { id: 2, isCorrect: true },
+      ],
+      [
+        { id: 1, isCorrect: false },
+        { id: 2, isCorrect: true },
+      ],
+      onSubmitFunction,
+    );
+
+    modalWindow.show(contemodalWindowContent);
+  }
+
+  function onSubmitFunction(playerName, playerResult, playerAnswersQuantity) {
+    modalWindow.close;
```

Comment:
Niedaleko jestes od sukcesu :) 
Ale czemu tutaj nie wywolujesz tej funkcji? 

Zapoznaj sie też z funkcja preventDefault() a propos event listenerow takich jak onClick. Tam widzialem, ze masz jakis problem z przeladowywaniem.

Reply:
teraz jest poprawione, ale trochę nie rozumiem po co tutaj mam zrobić preventDefault()? Przecież może się odpalać mouseEvent.
Naprawiłem komunikat z "Form submission canceled because the form is not connected" dodając do buttona `button.setAttribute('type', 'button');`, ale za to teraz zniknął komunikat o tym, że input is required i już nie wiem, że wiem jak to naprawić -.-

Reply:
Tzn. może masz lepszy pomysł, ale jak zrobisz, żeby nie przeładowywała się strona?

Reply:
modalWindow.close --> modalWindow.close() nie powoduje przeładowanie strony i wszystko działa poprawnie :-)

Reply:
Hmmm... Button nie ma type=submit jak w formsach i teraz nie mamy walidacji, że name jest wymagany. Dlatego też się nie przeładowuje, bo button submitujacy formularz robi to defaultowo. Wczesniej zdaje mi sie to miales :P ?

Reply:
Wcześniej button miał defaultowo type=submit i działała walidacja, ale wtedy po zamknięciu okna w konsoli wyrzucało ostrzeżenie "Form submission canceled because the form is not connected", dlatego dodałem `button.setAttribute('type', 'button');`, które problem rozwiązuje, ale z kolei nie ma wtedy walidacji... Pytanie które zło lepiej wybrać haha :D ? Albo jakoś inaczej walidację zrobić

Reply:
Raczej fajnie uzyc defaultowa walidacje, skoro juz jest :) 
A był button w formie? Jak nie to musi być przypisany:
`<button type="submit" form="form1" value="Submit">Submit</button>`
Tutaj masz form

Proponuje zrobic jakis komponent 
````
const FormSubmitButton = ({..., formId}) => {
...tutaj uzyc Buttona i dodac mu co trzeba. 
}
```

Reply:
Button jest wrzucony do forma, więc powinno być ok jeśli użyję metody render()

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.JavaScript.StarWarsQuiz/pull/72/commits/43f4205c1f9aebc9679eef1f56c1c6eaeabd1988 
ale pozostaje ten komunikat na żółto :-P i nie wiem czy może być, czy nie?

Reply:
Ta, to jest defaultowa walidacja, bym powiedział, że OK.